pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -29,7 +29,8 @@ import (
 var log = logf.Log.WithName("controller_clusterdeployment")
 
 const (
-	controllerName = "clusterdeployment"
+	controllerName                = "clusterdeployment"
+	ClusterDeploymentManagedLabel = "api.openshift.com/managed"
```

Comment:
fyi, no need for it to be a capitalized/exported variable since the test code is part of the same package and see the private variables, but not a big deal